docusaurus/docs/guides/deployments/10-linux/10-controller/60-backup.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Controller Backup
-sidebar_label: Controller
+title: Controller Backup and Recovery
```

Comment:
Why can't both the title and sidebar just be "Backup and Recovery". Your _in_ the controller leaf. "Controller" is redundant on the title

Reply:
The sidebar could say "Backup and Recovery." I've chosen aggressively short sidebar labels to preserve usability, especially for the sake of mobile. "Backup" isn't ambiguous and implies "Recovery." 

Re: title, Right, the title might seem redundant to someone that has only just navigated to that article via the sidebar, but there's no reason to think that's the case. I might follow a direct link to the article, and so the title should be meaningful without the sidebar or breadcrumb as context.